resources/qml/Toolbar.qml

```diff
@@ -67,7 +67,7 @@ Item
             }
         }
 
-        Item { height: UM.Theme.getSize("default_margin").height; width: 1; visible: extruders.count > 0 }
+        Item { height: UM.Theme.getSize("default_margin").height; width: UM.Theme.getSize("default_lining"); visible: extruders.count > 0 }
```

Comment:
This line causes the extruder tools to not have the correct top position. In order to fix it it should be changed to `UM.Theme.getSize("default_lining").width`. I've tested this locally but in order to simplify the merging process I'd be great if you can change that in your PR!

Reply:
Fixed, thanks